LLMonFHIR/FHIR Display/InspectResourceView.swift

```diff
@@ -115,10 +115,10 @@ struct InspectResourceView: View {
             do {
                 try await fhirResourceSummary.summarize(resource: resource, forceReload: forceReload)
                 loadingSummary = .idle
-            } catch let error as APIErrorResponse {
+            } catch let error as LLMError {
                 loadingSummary = .error(error)
             } catch {
-                loadingSummary = .error("Unknown error")
+                loadingSummary = .error("Unknown LLM processing error")
```

Comment:
Could we even use the unknown error type from SpeziLLM here and in other places?